dataflowengineoss/src/main/scala/io/shiftleft/dataflowengineoss/language/nodemethods/TrackingPointMethods.scala

```diff
@@ -28,4 +30,12 @@ class TrackingPointMethods[NodeType <: nodes.TrackingPoint](val node: NodeType)
       implicit context: EngineContext): Traversal[NodeType] =
     node.start.reachableBy(sourceTravs: _*)
 
+  def ddgIn(path: List[PathElement] = List())(implicit semantics: Semantics): Traversal[TrackingPoint] = {
```

Comment:
tracking points now offer `ddgIn` to walk one step back in the DDG and `ddgInPathElem` to retrieve the more detailed `PathElement` which, in particular, contains the edge label.